pallets/parachain-staking/src/lib.rs

```diff
@@ -1735,4 +1735,29 @@ pub mod pallet {
 			)
 		}
 	}
+
+	pub struct InitGenesisOnRuntimeUpgrade<T>(PhantomData<T>);
+	impl<T: Config> frame_support::traits::OnRuntimeUpgrade for InitGenesisOnRuntimeUpgrade<T> {
+		fn on_runtime_upgrade() -> frame_support::weights::Weight {
+			log::info!(target: "InitGenesisOnRuntimeUpgrade", "Initializing parachain-staking");
+
+			// Set collator commission to default config
+			<CollatorCommission<T>>::put(T::DefaultCollatorCommission::get());
```

Comment:
can these configs not be set by the config object?

Reply:
I'm not sure. For moonbeam the config values were stored as part of the genesis build (https://github.com/OAK-Foundation/moonbeam/blob/0eb967e9cb47e2b400c1011a7f9b77149b83054d/pallets/parachain-staking/src/lib.rs#L604) but since we're introducing the pallet in an upgrade I found that none of the config values were being stored/used until I added this runtime migration.